core/domain/story_jobs_one_off.py

```diff
@@ -131,3 +132,78 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class StoryMathRteAuditOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that checks for existence of math components in the skills."""
+
+    _LATEX_STRINGS_WITHOUT_SVG = 'latex-strings-without-svg'
+    _LATEX_STRINGS_HAVING_SVG = 'latex-strings-having-svg'
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [story_models.StoryModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+        story = story_fetchers.get_story_by_id(item.id)
+        html_string = ''
+        for node in story.story_contents.nodes:
+            html_string += node.outline
```

Comment:
'Story notes' is also an html field, right?

Reply:
Hmm yeah it can have HTML. I've added it now.
@seanlip , can pls re-run the `StoryMathRteAuditOneOffJob` just to be sure.

Reply:
Ran again, still None. @donosco98, feel free to merge this PR once CI passes!